ts/src/binance.ts

```diff
@@ -5067,9 +5067,7 @@ export default class binance extends Exchange {
         let type = this.safeString (transaction, 'type');
         if (type === undefined) {
             const txType = this.safeString (transaction, 'transactionType');
-            if (txType !== undefined) {
-                type = (txType === '0') ? 'deposit' : 'withdrawal';
-            }
+            type = (txType === '0') ? 'deposit' : 'withdrawal';
```

Comment:
@pcriadoperez can this be an alternative to your fix?

Reply:
Unfortunately not, as 'transactionType' can not exist in either the response of some of `fetchDeposits` and `fetchWithdrawals`. So I wouldn't recommend this as it would default to withdrawal incorrectly for some deposits.

As reference: a withdrawal:
```javascript
{
	"id": "cdb27985a2834be7a097208cba942dde",
	"amount": "0.0298",
	"transactionFee": "0.0002",
	"coin": "BTC",
	"status": 6,
	"address": "1Cd5xxxxxxxxxxxxxxxMM4HmC28F3N",
	"txId": "657044e282993af3xxxxxxxxxxxxxxxxxxx1008e9f2d67ed82d8482",
	"applyTime": "2023-03-21 22:49:42",
	"network": "BTC",
	"transferType": 0,
	"info": "bc1qm34lsc65zpw79lxxxxxxkqmk6ee3ewf0j77s3h",
	"confirmNo": 20,
	"walletType": 0,
	"txKey": ""
}
```
```javascript
{
	"id": "3328436891200069889",
	"amount": "30",
	"coin": "USDT",
	"network": "TRX",
	"status": 1,
	"address": "TRCP6KLTcoxxxxxxxxxxQGPwUsM5W",
	"addressTag": "",
	"txId": "9fa7b9d4a02xxxxxxxxxxxxxxxxxx811fc426a61d83d54f5bbc7ebb9fd261f12",
	"insertTime": 1676348699000,
	"transferType": 0,
	"confirmTimes": "1/1",
	"unlockConfirm": 0,
	"walletType": 0
}
```

Maybe we could use the presence of `insertTime` vs `applyTime` or `confirmTimes` vs confirmNo`